app/controllers/openid_connect/logout_controller.rb

```diff
@@ -25,6 +27,36 @@ def logout
       end
     end
 
+    # rubocop:disable Layout/LineLength
+    #
+    # +POST+ Handle logout request (with confirmation if initiated by relying partner)
+    # @see {OpenID Connect RP-Initiated Logout 1.0 Specification}[https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout]
+    # @note Response status code is <tt>307 Temporary Redirect</tt>
+    #   to preserve the method and body of the request
+    #   and to prevent non-conformant browsers from automatically
+    #   redirecting this route to {OpenidConnectLogoutForm#redirect_uri}
+    #   and skipping over the render of the logout confirmation page
+    #
+    # rubocop:enable Layout/LineLength
+    def create
+      @logout_form = build_logout_form
+      # @type [FormResponse]
```

Comment:
I am a huge proponent of YARD type annotations, but I feel like per-line ones of these might be a little noisy? especially since ideally we'd have them on the `#submit` method itself?

Reply:
I was trying keep the scope of the changes limited. I don't mind moving the annotation, but it would only impact `app/forms/openid_connect_logout_form.rb`, not any other form.

Reply:
Annotation moved. Added type for the `build_logout_form` method as well (`OpenIdConnectForm.new` returns an untyped object for some reason).